presto-native-execution/presto_cpp/main/common/Configs.cpp

```diff
@@ -157,6 +157,8 @@ SystemConfig::SystemConfig() {
           NUM_PROP(kDriverNumCpuThreadsHwMultiplier, 4.0),
           BOOL_PROP(kDriverThreadsBatchSchedulingEnabled, false),
           NUM_PROP(kDriverStuckOperatorThresholdMs, 30 * 60 * 1000),
+          NUM_PROP(
```

Comment:
Shall we set to 10 mins by default?

Reply:
kDriverCancelTasksWithStuckOperatorsThresholdMs should be larger than kDriverStuckOperatorThresholdMs.